doc/source/quickstart.rst

```diff
@@ -4,7 +4,8 @@ PDFmorph Tutorial
 #################
 
 Welcome! This will be a quick tutorial to accquaint users with PDFmorph
-and some of what it can do.
+and some of what it can do. For a more detailed tutorial, check out
+our :download:`user manual <pdfmorph.pdf>`.
```

Comment:
please see above.  no more texinfo so no more PDF version of the manual?   Do we need to talk again to make sure our goals are aligned?